### PR TITLE
Allow variables in paths and set default label.

### LINKED ISSUE #417: Make example configs useable out-of-the-box
screen shot 2016-06-21 at 11 38 56 am

### PR DESCRIPTION
**Pull request by matz-e**
I left the `lobster_` prefix for branding, and since @btovar is actually adjusting the `work_queue_status` display.  The first commit fixes the code, I'll add some more documentation in later commits.

Fixes #417.

### CODE REVIEW

**Comment by matz-e** [comment]
Now here's an additional idea: should we provide a `work_queue_factory` configuration in the `example/` directory?

**Comment by annawoodard** [comment]
Preferably one that matches one of the example configurations.

**Comment by matz-e** [comment]
The cool thing about sphinx is that we can just include/import the one from the examples directory into the docs :)

File: docs/run.rst [hunk 1]
```diff
@@ -53,7 +53,6 @@ The factory uses a config written in JSON, like:
 .. code-block:: json
 
     {
-      "master-name": "lobster_shuffle.*",
       "max-workers": 100,
       "min-workers": 0,
       "cores": 4,
```
Comment (annawoodard) [inline docs/run.rst]: Can we change the `memory` requirement to `4400` here to correspond to the `simple.py` configuration, then add the config to the examples and link from here?

After that I vote merge!

Comment (matz-e) [inline docs/run.rst]: Yes, we can. I'm going to comment some more on the ELK stuff, then return to this.

**Comment by annawoodard** [comment]
Thanks for doing this!

**Pull request merged**